src/icons/brand/blue/brand-blue.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../../../common/react/named-sfc';
+
+export const BrandBlue = NamedSFC('BrandBlue', () => (
+    <svg className="header-icon" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
My understanding is that the correct/standard-compliant equivalent of `alt=""` would be `role="presentation"` on the svg element (or if you wanted it to be non-presentational, to use `role="img"` and add a `<title>Accessibility Insights logo icon</title>` as the first child of the `<svg>` element).

Reply:
We don't want to announce the icon... that's why the original `<img>` tag had `alt=""`

Reply:
I wasn't able to remove auto-merge label before the bot completed this so, PR #703 have a fix for this comment